specification/cognitiveservices/Vision.ImageAnalysis/models.tsp

```diff
@@ -212,7 +180,7 @@ model ImageAnalysisResult {
 }
 
 @doc("The image metadata information such as height and width.")
-model ImageMetadataApiModel {
+model ImageMetadata {
```

Comment:
I only see the definition being renamed... Doesn't the use need to be also?

Reply:
Yes! Fixed and tested with sample code. Thanks!